README.md

```diff
@@ -143,3 +143,13 @@ For full PWA compatibility, you can put your [screenshots](https://developer.moz
 ```
 
 For beautiful screenshots, you can use [Progressier Screenshots Generator](https://progressier.com/pwa-screenshots-generator).
+
+### Cloudflare deployment
+
+**PNPM**
+1. Need set resolutions w/ `"sharp": "0.32.6"` on **package.json**
+2. It is necessary to create a file **npm-lock.yaml** by `pnpm install --no-frozen-lockfile`
```

Comment:
Let's say "re-run `pnpm install` to apply the changes" for point 2.

Would that work?

Reply:
I apologize, you are correct, only the `pnpm install` is sufficient here. The file **npm-lock.yaml** must exist.

Reply:
That'd be great, let me know. I'll merge this after you push the change.

Reply:
Im done.